doc/02_Pattern sicurezza/05_accesso-del-fruitore/03_id_auth_rest_01.rst

```diff
@@ -148,22 +148,23 @@ Esempio porzione JWT
 .. code-block:: python
 
    # header
-   
+
    {
    "alg": "ES256",
    "typ": "JWT",
    "x5c": [
    "MIICyzCCAbOgAwIBAgIEC..."
    ]
    }
-   
+
    # *payload*
    {
    "iat": 1554382877,
    "nbf": 1554382877,
    "exp": 1554382879,
    "iss": "https://api.fruitore.example",
    "sub": "https://api.fruitore.example",
+   "jti": "065259e8-8696-44d1-84c5-d3ce04c2f40d",
```

Comment:
@peppelinux in teoria sono d'accordo nel metterlo a prescindere che venga usato. Abbiamo però creato due profili nelle LG: uno che prevede l'enforcement del JTI ( ID_AUTH_REST_02) e questo senza enforcement (ID_AUTH_REST_01).

Quindi metterlo potrebbe confondere ;)

Reply:
K, faccio amend asap, ci vorrà pomeriggio. Denghiu

Reply:
done